website/docs/jellyfish/api/account.md

```diff
@@ -237,3 +237,26 @@ interface AccountHistoryCountOptions {
   no_rewards?: boolean
 }
 ```
+
+## sendTokensToAddress
+
+Creates a transfer transaction from your accounts balances.
+
+```ts title="client.account.sendTokensToAddress()"
+
+interface Account {
+  sendTokensToAddress(from: Address, to: Address, selectionMode: SelectionModeType = SelectionModeType.PIE): Promise<string>
```

Comment:
Again, there is a space between sendTokensToAddress and the rest. (Recommend: You can always compare this line with this line `return await this.client.call('sendtokenstoaddress', [from, to, selectionMode], 'number')` above)